common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -294,178 +309,124 @@ public FocusBarOverlay() {
         }
 
         @Override
-        protected void onConfigUpdate(ConfigHolder configHolder) {}
+        public String icon() {
+            return "۞";
+        }
 
-        protected void renderColoredBar(PoseStack poseStack, float renderY, CustomColor color, float progress) {
+        @Override
+        protected void renderBar(PoseStack poseStack, float renderY, float renderHeight, float progress) {
             Texture universalBarTexture = Texture.UNIVERSAL_BAR;
-            final float renderedHeight = universalBarTexture.height() / 2f * (this.getWidth() / 81);
-
             RenderUtils.drawColoredProgressBar(
                     poseStack,
                     universalBarTexture,
-                    color,
+                    textColor,
                     this.getRenderX(),
-                    renderY + 10,
+                    renderY,
                     this.getRenderX() + this.getWidth(),
-                    renderY + 10 + renderedHeight,
+                    renderY + renderHeight,
                     0,
                     0,
                     universalBarTexture.width(),
                     universalBarTexture.height(),
                     progress);
         }
 
-        protected void renderText(PoseStack poseStack, float renderY, String text) {
-            FontRenderer.getInstance()
-                    .renderAlignedTextInBox(
-                            poseStack,
-                            text,
-                            this.getRenderX(),
-                            this.getRenderX() + this.getWidth(),
-                            renderY,
-                            0,
-                            this.textColor,
-                            this.getRenderHorizontalAlignment(),
-                            this.textShadow);
+        @Override
+        public WynnBossBarUtils.BarProgress noProgress() {
+            return WynnBossBarUtils.NO_AWAKENED_PROGRESS;
         }
     }
 
-    public static class ManaBarOverlay extends Overlay {
-        @Config(key = "feature.wynntils.customBarsOverlay.overlay.manaBar.manaTexture")
-        public ManaTexture manaTexture = ManaTexture.a;
-
-        @Config(key = "feature.wynntils.customBarsOverlay.overlay.manaBar.textShadow")
-        public FontRenderer.TextShadow textShadow = FontRenderer.TextShadow.OUTLINE;
-
-        @Config(key = "feature.wynntils.customBarsOverlay.overlay.manaBar.flip")
-        public boolean flip = false;
-
-        @Config(key = "feature.wynntils.customBarsOverlay.overlay.manaBar.textColor")
-        public CustomColor textColor = CommonColors.LIGHT_BLUE;
-
-        public ManaBarOverlay() {
+    public static class FocusBarOverlay extends BaseBarOverlay {
+        public FocusBarOverlay() {
             super(
                     new OverlayPosition(
                             -30,
-                            52,
+                            -150,
```

Comment:
Why?

Reply:
I notice you changed positions for other overlays as well. What does this mean? Where are they now?

Reply:
This is because `BaseBarOverlay` was added, which then shifts all the classes in the .diff downward. In reality, there shouldn't be any changes to the positions.

Reply:
Oh, GH is so weird..